### PR TITLE
EC2 user data to work with assets

### PR DESCRIPTION
**Pull request by goruha**
## What
* Added user data to add assets make targets

## Why
* We want assets to be backuped

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,68 @@
+resource "null_resource" "default" {
+  triggers = {
+    id = "${lower(format("%v-%v-%v-%v", var.namespace, var.stage, var.name, "assets"))}"
```
Comment (osterman) [inline main.tf]: Why define a new `null_resource` when you can just use `tf_label` and append the specifier?

**Review by osterman — CHANGES_REQUESTED** [review]
See inline comment.

File: main.tf (hunk 1 above)
Comment (goruha) [reply main.tf]: It does not support 4 items in id

Comment (goruha) [reply main.tf]: It does not support 4 parts of id

Comment (osterman) [reply main.tf]: Overload the name. =)

Comment (osterman) [reply main.tf]: `name = "foo-something"`

Comment (osterman) [reply main.tf]: or better yet, let's add `attributes` as a new parameter to `tf_label` of type `list`. Use `join` to concatenate them with a `-`.

**Review by osterman — APPROVED** [review]

**Pull request merged**